packages/forms/src/directives/validators.ts

```diff
@@ -193,6 +193,95 @@ export class RequiredValidator implements Validator {
   }
 }
 
+export const MIN_VALIDATOR: Provider = {
+  provide: NG_VALIDATORS,
+  useExisting: forwardRef(() => MinValidator),
+  multi: true
+};
+
+/**
+ * A directive which installs the {@link MinValidator} for any `formControlName`,
+ * `formControl`, or control with `ngModel` that also has a `min` attribute.
+ *
+ * @experimental
+ */
+@Directive({
+  selector:
+      '[formControlName],input[type=number][min][formControl],input[type=number][min][ngModel]',
```

Comment:
I believe we should have the `input[type=number][min]` selector for the `[formControlName]` as well:
```suggestion
      'input[type=number][min][formControlName],input[type=number][min][formControl],input[type=number][min][ngModel]',
```